src/program/content/trakt.py

```diff
@@ -73,18 +76,33 @@ def missing(self):
             logger.log("TRAKT", "Trending fetching is disabled.")
         if not self.settings.fetch_popular:
             logger.log("TRAKT", "Popular fetching is disabled.")
+        if not self.settings.fetch_most_watched:
+            logger.log("TRAKT", "Most watched fetching is disabled.")
 
     def run(self):
         """Fetch media from Trakt and yield Movie, Show, or MediaItem instances."""
+        global last_update
```

Comment:
I was thinking a class attr, like `self.last_update` lol. I always though `global` was gross 😅

Reply:
👍🏻 Could you change it? Otherwise I'll do it in a few hours.